src/crypto/CMakeLists.txt

```diff
@@ -0,0 +1,105 @@
+# Copyright (c) 2023 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+if(ASM AND NOT MSVC)
+  # Check for SSE4.1 intrinsics.
+  set(SSE41_CXXFLAGS -msse4.1)
+  check_cxx_source_compiles_with_flags("${SSE41_CXXFLAGS}" "
+    #include <immintrin.h>
```

Comment:
For anybody else stumbling over this, removing `stdint.h` here in comparison to what's in the `configure.ac` is the right thing to do.